keps/sig-node/3953-node-resource-hot-plug/kep.yaml

```diff
@@ -0,0 +1,28 @@
+title: Node Resource Hot Plug
+kep-number: 3953
+authors:
+  - "@Karthik-K-N"
+  - "@mkumatag"
+  - "@kishen-v"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: provisional
+creation-date: 2023-10-04
+reviewers:
+  - "@smarterclayton"
+  - "@ffromani"
+  - "@SergeyKanzhelev"
+  - "@haircommander"
+approvers:
+  - "@sig-node-leads"
+see-also:
+
+stage: "alpha"
+
+latest-milestone: "v1.33"
+
+milestone:
+  alpha: ""
```

Comment:
Some information is still missing here e.g. feature gate and component. Please refer to https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml for the latest template. 

Also you need to add a file keps/prod-readiness/sig-node/3953.yaml so that this KEP is reviewed for Production Readiness. Refer to https://github.com/kubernetes/enhancements/tree/master/keps/prod-readiness/sig-node for examples. You have to mention a PRR approver from the list below in that file: https://github.com/kubernetes/enhancements/blob/3491aeec0a4bd47e697a54c041f331abe81f6ece/OWNERS_ALIASES#L194-L199

Reply:
Added. Thanks